### PR TITLE
Report package name and cwd during script errors

### PR DESCRIPTION
**Pull request by kirbysayshi**
Revisit of #102, but against rebirth. This is nearly a trivial change since rebirth appears to already have everything needed in context :sunglasses:. Not really sure how to test this! Suggestions?

### CODE REVIEW

File: src/run_cmd.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ export default function runCmd (cwd, argv) {
   }, function (err, statuses) {
     if (err) throw err
     const info = scripts.map((script, i) =>
-      script + ' exited with status ' + statuses[i]
+      pkg.name + ': '
```
Comment (alexanderGugel) [inline src/run_cmd.js]: We're now using babel, so we can safely use a template string here :sunglasses:

File: src/run_cmd.js [hunk 2]
```diff
@@ -37,7 +37,9 @@ export default function runCmd (cwd, argv) {
...
+      + script + ' exited with status ' + statuses[i] + ' '
+      + 'in directory ' + cwd
```
Comment (alexanderGugel) [inline src/run_cmd.js]: cwd will always be `process.cwd()`, which I think is expected. I don't think we have to log that.

**Comment by alexanderGugel** [comment]
Added some comments. Appreciate the PR :+1:

File: src/run_cmd.js (hunk 2 above)
Comment (kirbysayshi) [inline src/run_cmd.js]: I was assuming this file was also used when running lifecycle scripts of dependencies. Is that not the case?

And is package.json always the root package.json then?

**Comment by kirbysayshi** [comment]
I was revisiting this tonight, have a few questions (above). Seems like most of this is in the codebase already! But I still think it would be useful to log the name of the package in the case of an error, unless I'm misunderstanding how this code is executed.

**Comment by alexanderGugel** [comment]
This is already in the code base.

**Pull request closed without merging**